seednode/src/main/java/bisq/seednode/SeedNodeMain.java

```diff
@@ -53,7 +53,6 @@ public static void main(String[] args) throws Exception {
     protected void doExecute(OptionSet options) {
         super.doExecute(options);
 
-        checkMemory(bisqEnvironment, this);
```

Comment:
maybe I missed this in a discussion somewhere, but wat is the motivation to remove this check?

Reply:
Please discuss with @freimair If it is removed it might require that nodes run as systemd service to guarantee a restart in case of a out of memory. I would suggest that just a few nodes use that first to not risk that all fail if there is an issue.

Reply:
The thing is, the checkMemory routine attempts to restart the seed node and does so by doing highly specific stuff. I had my seed node run with systemd and everytime the checkMemory did its thing, I had to manually restore and restart the node. Even without systemd, the restart did not work.
Restarting a system service after it fails due to external stuff (memory in this case) is not to be implemented in the service itself. That only complicates things and hardly works on a wider range of servers.

Reply:
thx for the reply, I don't disagree with the change but seednode operators will have to be notified and as Manfred says maybe a staggered deploy would be best.